app/Resources/views/page/adherer.html.twig

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html.twig' %}
+
```

Comment:
Duplicate of `https://github.com/EnMarche/en-marche.fr/blob/master/app/Resources/views/membership/register.html.twig`?